ui/tutorial-renderer/src/main/res/layout/tutorial_tips_learn.xml

```diff
@@ -1,36 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
+   >
 
     <data>
         <import type="org.cru.godtools.tutorial.activity.TutorialActivity" />
 
-        <variable name="callbacks" type="org.cru.godtools.tutorial.TutorialCallbacks" />
+
         <variable name="formatArgs" type="android.os.Bundle" />
     </data>
```

Comment:
you can also remove this `<data>` xml since it's not used anywhere. I'm not going to hold the PR up on this change though.